css-selectors/selectorsList.js

```diff
@@ -0,0 +1,10 @@
+const titleSelector = 'title#hgvdhet_';
```

Comment:
I have added this id hgvdhet_ especially to show how we can face autogenerated selectors. Better to not use it in real projects.